src/uct/sm/mm/posix/mm_posix.c

```diff
@@ -141,8 +141,12 @@ uct_posix_md_query(uct_md_h tl_md, uct_md_attr_v2_t *md_attr)
     }
 
     shm_size = shm_statvfs.f_bsize * shm_statvfs.f_bavail;
-    uct_mm_md_query(&md->super, md_attr,
-                    (shm_size < posix_config->shm_min_size) ? 0 : shm_size);
+    if (shm_size < posix_config->shm_min_size) {
+        ucs_trace("md alloc disabled: only %zu bytes left in shm", shm_size);
```

Comment:
maybe it can even be a debug trace, because it is not supposed to be invoked frequintly

Reply:
ok for me, was putting trace to err on the safe side since this is called on the mem alloc for md method code path, @yosefe what do you think?

Reply:
should be ucs_debug because it is enabled in release mode

Reply:
fixed

Reply:
force-pushed for this simple one.